Validation/Geometry/test/runP_InnerServices_cfg.py

```diff
@@ -6,9 +6,10 @@
 
 #Geometry
 #
-process.load("Geometry.CMSCommonData.cmsSimIdealGeometryXML_cfi")
-
+process.load("Geometry.CMSCommonData.cmsExtendedGeometry2017XML_cfi")
 process.load("Geometry.TrackerNumberingBuilder.trackerNumberingGeometry_cfi")
+process.load("Geometry.HcalCommonData.hcalParameters_cfi")
```

Comment:
Good point, it would have been more efficient just to use the full geometry cffs. Since it is important for this pull request to go through so that the functional material budget validation package can become officially available, I propose to make this simple fix (which has essentially a cosmetic effect and doesn't affect the functioning of the code itself) in a separate pull request.